src/NuGet.Licenses/Views/License/DisplayLicense.cshtml

```diff
@@ -5,4 +5,10 @@
 
 <h2>@Model.Title</h2>
 
-@Model.Text
\ No newline at end of file
+<pre>@Model.Text </pre>
+<style>
```

Comment:
Please remove the inline style. The target branch now has `Site.css` you can update it instead.

Reply:
Delete it